src/styles/index.scss

```diff
@@ -1,3 +1,17 @@
-body {
+$mainColor: #060b35;
+$secondaryColor: #616070;
+$buttonColor: #00acdc;
+$borderColor: #f3f3f3;
+$fontFamily: "Roboto", sans-serif;
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+html {
+  font-family: $fontFamily;
+}
+
+@import "blocks/card.scss";
+@import "blocks/stars.scss";
```

Comment:
it is better to define the import at the top of the file

Reply:
done